### PR TITLE
Fix up .kitchen.docker.yml to use dokken

### PR DESCRIPTION
**Pull request by someara**
- Fixing missing suites section in .kitchen.docker.yml
- Updating to use kitchen-dokken for speed and cleaner Travis output

### CODE REVIEW

File: .kitchen.docker.yml [hunk 1]
```diff
@@ -1,21 +1,104 @@
...
+  driver:
+    image: centos:7
+    intermediate_instructions:
+      - RUN yum clean all
+      - RUN yum -y install net-tools lsof
+    pid_one_command: /usr/lib/systemd/systemd
+
+- name: fedora-21
+  driver:
+    image: fedora:21
```
Comment (tas50) [inline .kitchen.docker.yml]: We should make this 23 since 21 is already EOL

File: .kitchen.docker.yml [hunk 2]
```diff
@@ -1,21 +1,104 @@
...
+
+- name: ubuntu-14.04
+  driver:
+    image: ubuntu-upstart:14.04
+    pid_one_command: /sbin/init
+    intermediate_instructions:
+      - RUN /usr/bin/apt-get update
+      - RUN /usr/bin/apt-get -y install apt-transport-https net-tools
+
+- name: ubuntu-15.04
```
Comment (tas50) [inline .kitchen.docker.yml]: 15.04 is EOL so this should go

File: .kitchen.docker.yml [hunk 3]
```diff
@@ -1,21 +1,104 @@
...
+  driver:
+    image: ubuntu:15.04
```
Comment (tas50) [inline .kitchen.docker.yml]: We should skip right to 16.04 to get systemd and a LTS

**Comment by someara** [comment]
Removed the oldbuntu's..... this is turning out to be a bit messy.
I'm using PR to trigger travis builds... don't merge this yet. The yaks run deep here.

**Comment by tas50** [comment]
This needs to get rebased before it can get merged @someara

**Comment by tas50** [comment]
We should probably add dokken at some point, but it's easier to start over at this point.

**Pull request closed without merging**